src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return {
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
+    ownerCount: data.OwnerCount,
+    previousTransactionID: data.PreviousTxnID,
+    previousTransactionLedgerVersion: data.PreviousTxnLgrSeq
```

Comment:
Ah, yes: "Note: The deprecated PreviousTxnID transaction parameter was removed entirely in rippled 0.28.0. Use AccountTxnID instead." - https://ripple.com/build/transactions/#common-fields